test/integration/dockerregistryclient_test.go

```diff
@@ -224,14 +222,19 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 	}
 }
 
+func TestRegistryClientImageV2ManifestV2(t *testing.T) {
+	t.Skip("openshift/schema-v2-test-repo is being pushed by Docker 1.10, which causes this test to fail because digests don't match")
```

Comment:
I really like @miminar idea here. It's not a hack, it's a test and we need to make sure we work ok. 

Where is the fix for v2 schema client? If that works the test should as well, right?